Update Beginner-friendly coding meetups

Nirvikalpa108 opened the pull request:
I deleted a few meetups that haven't met in over a year and I updated Founders & Coders.

Nirvikalpa108 committed:
Update Beginner-friendly coding meetups

I deleted a few meetups that haven't met in over a year and I updated Founders & Coders.

JennyBrennan commented on index.html:
```diff
@@ -71,9 +71,7 @@ <h2 id="technology">Technology-specific</h2>
-      <li><a href="https://www.facebook.com/groups/KeepCalmAndCarryOnCoding/" title="Keep Calm and Carry on Coding facebook group">Keep Calm and Carry on Coding</a> - a fortnightly meetup aimed at anyone have who has recently done a coding course and wants to continue learning.</li>
-      <li><a href="http://codeandcake.club">Code &amp;&amp; Cake</a> - regular meetups where you can work on your own coding side projects whilst eating cake (hurrah!); open to all experience levels.</li>
-      <li><a href="http://www.meetup.com/founderscoders/events/225965453/" title="Founders and Coders Coding for Everyone">Coding for Everyone (Founders &amp; Coders)</a> - find someone of a similar level to you to talk through and work on code with.</li>
+      <li><a href="https://www.meetup.com/founderscoders/" title="Founders and Coders Coding for Women">Coding for Women (Founders &amp; Coders)</a> - find someone of a similar level to you to talk through and work on code with.</li>
```
Suggest we take out the title attribute here - it doesn't add additional information for the user; sorry that the example you had to go on also had an unnecessary title attribute!

```suggestion
      <li><a href="https://www.meetup.com/founderscoders/">Coding for Women (Founders &amp; Coders)</a> - find someone of a similar level to you to talk through and work on code with.</li>
```

JennyBrennan replied:
Based on the discussion in issue #8, could we replace line 83 with this Coding for Women link & description instead? Thank you! 👍

JennyBrennan committed:
remove title attribute

Co-Authored-By: Nirvikalpa108 <amina.adewusi@gmail.com>

Nirvikalpa108 commented:
Great, I've accepted your changes (removed the title attribute and added the Coding for Women link and description). 

I hope I've done this correctly? Let me know if not! Thank you! 😃

Nirvikalpa108 committed:
Remove Founders&Coders from Beginners section 

Given our conversation about keeping meetups for underrepresented groups in that category and nowhere else, I've deleted the duplicate founders & coders from the beginners section and left the one that was in the u/r groups section. I hope that's ok! Let me know if not? Thank you soo much!

JennyBrennan commented:
Awesome - merging. Thanks so much for your work on this!
